### PR TITLE
Move testbench package to dev dependencies

### PR DESCRIPTION
**Pull request by juukie**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,11 +17,11 @@
     ],
     "require": {
         "php": "^7.0",
-        "illuminate/support": "~5.5.0",
-        "orchestra/testbench": "~3.5.0",
+        "illuminate/support": "~5.5.0"
```
Comment (freekmurze) [inline composer.json]: Add a comma here to fix the tests.

**Comment by freekmurze** [comment]
Good catch, thanks! Small issue with that missing comma, could you fix that?

**Comment by juukie** [comment]
Sorry @freekmurze, how could I miss that stupid comma :S.
I've added it.

**Comment by freekmurze** [comment]
Thank you 👍

**Pull request merged**